### PR TITLE
Add Nitro Bypass && MessageEvents

### PR DESCRIPTION
**Pull request by ArjixWasTaken**

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
LGTM, dunno about you

File: src/plugins/nitroBypass.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export default definePlugin({
+    name: "Nitro Bypass",
+    author: "ArjixWasTaken",
+    description: "Allows you to stream in nitro quality and send fake emojis.",
+    patches: [
+        ...[
+            "canUseAnimatedEmojis",
+            "canUseEmojisEverywhere",
+            "canUseHigherFramerate"
+        ].map(func => {
```
Comment (Vendicated) [inline src/plugins/nitroBypass.ts:14]: Assuming these are all the same module, move this map below to replacement
replacement: ["canUse..."].map(....)

File: src/patcher.ts [hunk 2]
```diff
@@ -3,8 +3,10 @@ import electron, { app, BrowserWindowConstructorOptions } from "electron";
 import installExt, { REACT_DEVELOPER_TOOLS } from "electron-devtools-installer";
 import { join } from "path";
 import { initIpc } from './ipcMain';
+import Logger from "./utils/logger";
 
-console.log("[Vencord] Starting up...");
+const logger = new Logger("Patcher", "#700b90")
```
Comment (Vendicated) [inline src/patcher.ts:8]: does the console even support styled logging?

Comment (ArjixWasTaken) [reply src/patcher.ts:8]: The nodejs console? It should

**Comment by Vendicated** [comment]
Oh that went to the wrong branch, can you reopen a PR to the main branch please?

**Pull request merged**